Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/OperationMode.mo

```diff
@@ -559,6 +532,66 @@ equation
   connect(not1.y, winOpe.u)
     annotation (Line(points={{322,50},{338,50}}, color={255,0,255}));
 
```

Comment:
remove this empty line and have a empty line between this section and info section.

Reply:
Changed